handlers.go

```diff
@@ -1029,6 +1033,8 @@ func GetFromGameHandler(c echo.Context) error {
 	}
 	pageHTML, _ := bot.GetPageContent(vals)
 	pageHTML = replaceHostname(bot, pageHTML)
+	pageHTML = removeCookieBanners(pageHTML)
```

Comment:
Remove the cookie Banner from HTML.
![image](https://user-images.githubusercontent.com/21330562/131993536-1fede11c-eb41-45d3-bc9c-320cdef07695.png)